projects/igniteui-angular/src/lib/select/select.component.spec.ts

```diff
@@ -200,6 +200,17 @@ describe('igxSelect', () => {
             fixture.detectChanges();
             expect(select.collapsed).toBeTruthy();
         }));
+        it('should close dropdown on clicking selected item', fakeAsync(() => {
```

Comment:
We may add an additional "iteration" - opening and clicking again the selected item through the UI.

Reply:
done